Documentation/op-guide/security.md

```diff
@@ -32,6 +34,8 @@ The peer options work the same way as the client-to-server options:
 
 `--peer-trusted-ca-file=<path>`: Trusted certificate authority.
 
+`--peer-auto-tls`: Generate and use self-signed certificates for SSL/TLS connections between peers.
```

Comment:
Use auto generated and self-signed certificates for TLS connections between peers.